### DISCUSSION TITLE
Adding Language Files to AzuraCast for Complete Translation

### DISCUSSION THREAD

**Discussion by BornKillaz** [AzuraCast/AzuraCast#7033] [Q&A]
Hi!

I would like to know if there is a way to include a language file in AzuraCast. 

The reason is that in my language, only a small percentage is translated, and I need a complete translation. 

Is there any way to add language files to the AzuraCast installation?"

**Comment by LordHelmchen666** [comment]
You could help translate Azuracast here 
https://www.azuracast.com/docs/contribute/translation/
https://translate.azuracast.com/

**Comment by BornKillaz** [comment]
Thank you for your response, @LordHelmchen666. I've already contributed to the translation, but I need a quicker solution. Waiting for my contribution to be approved and included in future updates isn't feasible for me right now.

While I'm willing to do a complete translation for AzuraCast, I also need an immediate solution. I'm wondering if there's any way to use our own translation files in the meantime.
